### PR TITLE
12.0 backport module change auto install

### PR DESCRIPTION
**Pull request by legalsylvain**
- Trivial backport. (nothing to do, see https://github.com/OCA/server-tools/pull/2092/commits/cfd2f36b39cd9ec147e8c84242654889cc382db8)

- Used in production since several monthes with the following configuration :
```
modules_auto_install_disabled = 
	snailmail,
	iap,
	mail_bot,
	partner_autocomplete
```

- Used locally with the following configuration : 
```
modules_auto_install_enabled = 
	base_technical_features,
	disable_odoo_online,
	partner_disable_gravatar,
	mail,
	web,
	web_no_bubble,
	web_refresher,
	web_responsive,
	web_responsive_company,
	web_tree_dynamic_colored_field
```

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi I forgot this draft PR that is used in production since monthes.
It is ready to review. 
CC : @moylop260, @marcelsavegnago, @fcayre, @quentinDupont

File: module_change_auto_install/patch.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    []
+    """
+    s = (s or "").strip(" ,")
+    if not s:
+        return []
+    return [x.strip() for x in s.split(",")]
+
+
+def _overload_load_information_from_description_file(module, mod_path=None):
+    res = _original_load_information_from_description_file(module, mod_path=None)
```
Comment (fcayre) [inline module_change_auto_install/patch.py:35]: Should be mod_path=mod_path here no ?

**Review by fcayre — CHANGES_REQUESTED** [review]

**Review by thomaspaulb — APPROVED** [review]
Approved since the same bug is also in the 14.0 version, but still it should be fixed I think

**Pull request merged**